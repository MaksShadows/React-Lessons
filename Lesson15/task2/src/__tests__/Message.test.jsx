import React from 'react';
import {shallow} from 'enzyme';
import Message from '../Message'

describe('<Message />', () => {
    it('sould return message with text', () => {
        const message = 'Hello React';
        const wrappedComponent = shallow(<Message message={message} />);

        expect(wrappedComponent.find('.message').exists()).toEqual(false);
    });

    it('sould return empty message', () => {
        const message = '';
        const wrappedComponent = shallow(<Message message={message} />);

        expect(wrappedComponent.find('.message').exists()).toEqual(false);
    });
});
