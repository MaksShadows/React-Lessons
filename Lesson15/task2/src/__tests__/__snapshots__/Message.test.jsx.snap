// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> sould return empty message 1`] = `""`;

exports[`<Message /> sould return message with text 1`] = `""`;
